python_loops.py

```diff
@@ -0,0 +1,79 @@
+alphabet = 'abcdefghijklmnopqrstuvwxyz' # Ex. 9
+zen = """
+Beautiful is better than ugly.
+Explicit is better than implicit.
+Simple is better than complex.
+Complex is better than complicated.
+Flat is better than nested.
+Sparse is better than dense.
+Readability counts.
+Special cases aren't special enough to break the rules.
+Although practicality beats purity.
+Errors should never pass silently.
+Unless explicitly silenced.
+In the face of ambiguity, refuse the temptation to guess.
+There should be one-- and preferably only one --obvious way to do it.
+Although that way may not be obvious at first unless you're Dutch.
+Now is better than never.
+Although never is often better than *right* now.
+If the implementation is hard to explain, it's a bad idea.
+If the implementation is easy to explain, it may be a good idea.
+Namespaces are one honking great idea -- let's do more of those!
+"""
+set(alphabet).difference(set(zen))
+
+d = {'payton':'An interpreted, object-oriented programming language'} # Ex. 10-a
+d["python"] = d.pop("payton")
+
+dictionary = {("name", "surname"):("telephone_number")} # Ex. 10-b
+
+info = {('Name', 'Surname'):('John', 'Doe')} # Ex. 11
+print("_".join(info[("Name", "Surname")]))
+
+names_list = ['Jiri', 'Jan', 'Marie', 'Petr', 'Jana', 'Josef', 'Pavel', 'Martin', 'Jaroslav', 'Tomas', 'Eva', 'Miroslav', 'Hana', 'Anna', 'Zdenek', 'Frantisek', 'Vaclav', 'Michal', 'Lenka', 'Katerina'] # Ex. 1
+name = input()
+if name in names_list:
+    print("reply")
+else:
+    print("another reply")
+    
+spelling_alphabet = {'a':'alfa', 'b':'bravo', 'c':'charlie', 'd':'delta', 'e':'echo', 'f':'foxtrot', 'g':'golf', 'h':'hotel', 'i':'india', 'j':'juliett', 'k':'kilo', 'l':'lima', 'm':'mike', 'n':'november', 'o':'oscar', 'p':'papa', 'q':'quebec', 'r':'romeo', 's':'sierra', 't':'tango', 'u':'uniform', 'v':'victor', 'w':'whiskey', 'x':'x-ray', 'y':'yankee', 'z':'zulu'} # Ex. 2
+user_name = list(input()[::].lower())
+for letter in user_name:
+    for key, value in spelling_alphabet.items():
+        if letter is key:
+            print(value)
+        else:
+            continue
+
+a = [[1,2,3], # Ex. 3-loops

```

Comment:
Male doporuceni, muzete pomoci mezer udelat kod prehlednejsi. Pokud tam jsou zavorky, tak to bude pythonu fuk kolik je tam prazdneho prostoru. Tedy:

```
x = [[a, b, c,]
     [a, b, c,]]
```

misto 

```
x = [[a, b, c,]
[a, b, c,]]
```